webapp/src/pages/Settings.tsx

```diff
@@ -716,11 +759,11 @@ const Settings: React.FC = () => {
             disabled={isUpdatingConfig}
             onClick={() => updateConfig({ jobId, body: partialConfig })}
```

Comment:
Instead of `isUpdateSuccessful`, which is a little bit complicated with the `useEffect`, you can directly do this:
```suggestion
            onClick={() =>
              updateConfig({ jobId, body: partialConfig })
                .unwrap()
                .then(onClose)
            }
```

Reply:
okay, cool. Added the suggestion.